Polyhedron/demo/Polyhedron/Plugins/Convex_decomposition/Nef_plugin.cpp

```diff
@@ -56,6 +56,7 @@ void init(QMainWindow* mw,
       action->setProperty("subMenuName",
                           "Boolean Operations");
     _actions<< actionConvexDecomposition;
+    connect(actionConvexDecomposition, SIGNAL(triggered()), this, SLOT(on_actionConvexDecomposition_triggered()));
```

Comment:
The nef_plugin does not inherit from the Polyhedron_demo_plugin_helper, so it does not have access to the auto-connect function. I admit I did not look for the reason it is not, as it was far easier and quicker to just add the missing connect than to make it inherit form the helper